src/NUnitFramework/framework/Internal/PlatformHelper.cs

```diff
@@ -87,13 +88,9 @@ public PlatformHelper( OSPlatform os, RuntimeFramework rt )
         /// <param name="platforms"></param>
         /// <returns></returns>
         public bool IsPlatformSupported( string[] platforms )
-        {
-            foreach( string platform in platforms )
-                if ( IsPlatformSupported( platform ) )
-                    return true;
-
-            return false;
-        }
+            {
+            return platforms.Any (IsPlatformSupported);
```

Comment:
Linq for the win! :wink: 

You can't tell me this isn't more readable and an improvement...